ReadMe.md

```diff
@@ -80,7 +79,7 @@ To register the GDExtension library with Godot, you need to create two files rel
 We highly recommend to have a look at a working example in the `examples/dodge-the-creeps` directory.
 This integrates a small game with Godot and has all the necessary steps set up.
 
-API documentation can be generated locally using `cargo doc -p godot --no-deps --open`.
+API documentation can be generated locally using `./check.sh dok` (or `./check doc` to refresh an opened page).
```

Comment:
```suggestion
API documentation can be generated locally using `./check.sh dok` (or `./check.sh doc` to refresh an opened page).
```

Might not be the place for discussing it, but just mentioning "dok" as a key is prone to being considered a typo IMO. Maybe we should make it clear (here?) that "dok" (sic!) is meant

Reply:
Thanks! Yeah, originally there was just `doc` and I found myself always wanting to open the browser. Maybe swapping the sentence in the parentheses with the one outside could clarify this.